src/Og.php

```diff
@@ -17,6 +17,137 @@
 class Og {
 
   /**
+   * Static cache for groups per entity.
+   *
+   * @var array
+   */
+  protected static $entityGroupCache = [];
+
+  /**
+   * Create an organic groups field in a bundle.
+   *
+   * @param $field_name
+   *   The field name.
+   * @param $entity_type
+   *   The entity type.
+   * @param $bundle
+   *   The bundle name.
+   */
+  public static function createField($field_name, $entity_type, $bundle) {
+    $og_field = static::fieldInfo($field_name)
+      ->setEntityType($entity_type)
+      ->setBundle($bundle);
+
+    if (!FieldStorageConfig::loadByName($entity_type, $field_name)) {
+      $og_field->fieldDefinition()->save();
+    }
+
+    // Allow overriding the field name.
+    // todo: ask if we need this.
+//    $og_field['field']['field_name'] = $field_name;
```

Comment:
yeah, this is handy, when we want to use the config of a field name (e.g. `OG_AUDIENCE_FIELD`, but override some config, such as the label, required, etc').